### PR TITLE
Output list in template order.

### PR DESCRIPTION
**Pull request by Sowevo**
Output list in template order.

use `-s template-order`

### CODE REVIEW

File: iptvtools/models.py [hunk 1]
```diff
@@ -158,7 +161,7 @@ def filter(self):
             pbar.write(f'{url}, {status}!')
 
     def __custom_sort(self, url):
-        """Sort by tvg-id, resolution and title."""
+        """Sort by tvg-id, resolution,template-order and title."""
```
Comment (huxuan) [inline iptvtools/models.py]: Add a space before `template-order`?

**Comment by huxuan** [comment]
Hi @Sowevo, Thanks for the contribution! Just a minor comment to make the docstring lint. Just let me know if you have any concern.

**Comment by Sowevo** [comment]
ok,i modified it.

**Comment by huxuan** [comment]
> ok,i modified it.

Thanks again for the contribution. It is already release with v0.2.8 [1].

[1] https://pypi.org/project/iptvtools/0.2.8/

**Pull request merged**